cmd/utils/flags.go

```diff
@@ -610,6 +610,21 @@ var (
 		Usage: "InfluxDB `host` tag attached to all measurements",
 		Value: "localhost",
 	}
+
+	EWASMFlag = cli.BoolFlag{
+		Name:  "ewasm",
+		Usage: "Enable eWASM capabilities on this chain",
+	}
+	EWASMInterpreterFlag = cli.StringFlag{
+		Name:  "ewasm.interpreter",
+		Usage: "Path to the ewasm interpreter",
+		Value: "wagon",
+	}
+	EVMInterpreterFlag = cli.StringFlag{
+		Name:  "evm.interpreter",
+		Usage: "Path to the evm interpreter",
+		Value: "geth",
```

Comment:
Please name these `--vm.evm` and `--vm.ewasm`. The defaults should be empty, which geth can interpret by running the built in VMs.

Reply:
Also add `(default = built in implementation)`